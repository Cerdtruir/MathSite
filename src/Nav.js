import {
  Link,
} from 'react-router-dom';

const nav = () => (
  <nav>
    <h1>Math Site</h1>
    <ul className="navUl">
      <li>
        <Link to="/">Home</Link>
      </li>
      <li>
        <Link to="/calculator">Calculator</Link>
      </li>
      <li>
        <Link to="/quote">Quote</Link>
      </li>
    </ul>
  </nav>
);

export default nav;
