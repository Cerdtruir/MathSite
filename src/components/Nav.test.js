import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from './Nav';

it('Nav renders correctly', () => {
  const tree = renderer
    .create(<BrowserRouter><Nav /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
