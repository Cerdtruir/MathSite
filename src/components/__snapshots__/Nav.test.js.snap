// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<nav>
  <h1>
    Math Site
  </h1>
  <ul
    className="navUl"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
